packages/aws-cdk/test/integ/cli/test-cdk-termination-protection.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+source ${scriptdir}/common.bash
+# ----------------------------------------------------------
+
+setup
+
+stack="${STACK_NAME_PREFIX}-termination-protection"
```

Comment:
might also want to add this to the [cleanup](https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/test/integ/cli/common.bash#L91-L101) script so we're handling it proactively